webdriverio/web/examples/typescript/test/pageobjects/login.page.ts

```diff
@@ -0,0 +1,23 @@
+import Page from './page';
+
+/**
+ * sub page containing specific selectors and methods for a specific page
+ */
+export class LoginPage extends Page {
```

Comment:
You now need to instantiate the class every time  you use it. Why not do something like in our BP examples of WebdriverIO where you have

```ts
class LoginPage extends Page {
  // ...
}

export default new LoginPage();
```

Then you can easily `import` the `LoginPage` in your specs and use it like this

```ts
import LoginPage from '../page-objects/LoginPage';

describe('My Login application', () => {
    it('should login with valid credentials', () => {
        // ...
        //act
        LoginPage.open();
});
```

Reply:
@wswebcreation excellent suggestion, thanks